src/spans/awsSpan.js

```diff
@@ -239,13 +245,13 @@ export const getHttpInfo = (requestData, responseData) => {
   return { host, request, response };
 };
 
-export const getBasicChildSpan = (spanId, spanType) => {
+export const getBasicChildSpan = (transactionId, awsRequestId, spanId, spanType) => {
```

Comment:
Maybe this function should calculate `transactionId` and `awsRequestId` by itself?
It will make the other code simpler and anyway this function access global variables (`TracerGlobals.getHandlerInputs()`).
I wrote a similar comment at the beginning.

Reply:
Its need to be calculated when the request is fired, there is a big impact to when the calculation is made, its cant be calculated inside getBasicChildSpan because this function called when the request ends